### PR TITLE
rp: fix redeclare error

### PR DESCRIPTION
**Pull request by dunn**
otherwise i'm seeing
```
Can't redeclare "my" in "my" at /var/www/html/cgi-bin/rp.pm line 89, near ", my"
syntax error at /var/www/html/cgi-bin/rp.pm line 228, near "TransitiveClosure ["
Global symbol "$cycle" requires explicit package name (did you forget to declare "my $cycle"?) at /var/www/html/cgi-bin/rp.pm line 230.
Global symbol "$cycle" requires explicit package name (did you forget to declare "my $cycle"?) at /var/www/html/cgi-bin/rp.pm line 235.
Global symbol "$cycle" requires explicit package name (did you forget to declare "my $cycle"?) at /var/www/html/cgi-bin/rp.pm line 242.
syntax error at /var/www/html/cgi-bin/rp.pm line 242, near "TransitiveClosure ["
Global symbol "$cycle" requires explicit package name (did you forget to declare "my $cycle"?) at /var/www/html/cgi-bin/rp.pm line 243.
syntax error at /var/www/html/cgi-bin/rp.pm line 383, near "TransitiveClosure ["
Global symbol "$cycle" requires explicit package name (did you forget to declare "my $cycle"?) at /var/www/html/cgi-bin/rp.pm line 384.
Compilation failed in require at /var/www/html/cgi-bin/results.pl line 19.
BEGIN failed--compilation aborted at /var/www/html/cgi-bin/results.pl line 19.
```

### CODE REVIEW

File: cgi-bin/rp.pm [hunk 1]
```diff
@@ -1,414 +1,414 @@
...
+}
+
+# TransitiveClosure(m, winner, loser, n) adds the preference
+# for winner over loser to the nxn matrix m and computes the
+# transitive closure (in m). If this creates any new cycles,
+# returns 1, otherwise 0. It uses a worklist algorithm because
+# that is faster than a full Floyd-Warshall when the matrix
+# is already pretty much done.
+sub TransitiveClosure {
+    my ($mref, $winner, $loser, $num_choices) = @_;
```
Comment (dunn) [inline cgi-bin/rp.pm:89]: sorry, the diff isn't very useful because my editor decided to also correct the DOS line endings; here's the relevant change

Comment (andrewcmyers) [reply cgi-bin/rp.pm:89]: This seems like a good idea but would you mind undoing the line terminator changes so the change is localized?

Comment (dunn) [reply cgi-bin/rp.pm:89]: sure, no problem

**Review by andrewcmyers — APPROVED** [review]
thanks!

**Pull request merged**